overlord/fdestate/backend/seal.go

```diff
@@ -112,12 +112,12 @@ func sealRunObjectKeys(key secboot.BootstrappedContainer, pbc boot.PredictableBo
 	// path only unseals one object because unsealing is expensive.
 	// Furthermore, the run object key is stored on ubuntu-boot so that we do not
 	// need to continually write/read keys from ubuntu-seed.
-	primaryKey, err := secbootSealKeys(runKeySealRequests(key, useTokens), sealKeyParams)
+	createdPrimaryKey, err := secbootSealKeys(runKeySealRequests(key, useTokens), sealKeyParams)
```

Comment:
Agreed, according to `sbNewProtectedKey`, it will get created only if nil was passed as the primary key

Reply:
It might be created, or the same if not nil. If we are in the case where use tokens, the primary key will be already picked because of the plainkey, and the parameter `primaryKey` will contain that key. And `createdPrimaryKey` will be the same key. If we do not use a tokens, then we did not chose a primary key yet. Then `primaryKey` is nil. And then `createdPrimaryKey` is a new key.

I could name them "primaryKeyInput" and "primaryKeyOutput". Or I could name same "primaryKey", "chosenPrimaryKey". Or "maybePrimaryKey" and "primaryKey".

Reply:
maybePrimaryKey and primaryKey seem fine here

Reply:
Done.